Processing/Explore_New_Parks.Rmd

```diff
@@ -0,0 +1,303 @@
+---
+title: "Data_Availability_Parks"
+author: "KEC"
+date: "2024-11-05"
+output: word_document
+editor_options: 
+  chunk_output_type: inline
+---
+
+Define park units that you want to explore data for
+
+```{r}
+
+parks <- c("VALL","CHIR", "FOBO", "CORO", "CACH", "HUTR", "NAVA", "ZION", "CEBR")
+```
+
+
+```{r setup, include=FALSE}
+source("setup.R")
+
+pal <- c("black",
+            "#dca237", 
+    "#6eb2e4", 
+   "#459a74", 
+  "#eee461", 
+   "#3070ad", 
+  "#c66426", 
+   "#c07da5",
+  "#ECD8E4"
+)
+
+```
+
+# Input water supply table points
+
+```{r}
+
+table_path <- "data/Water_Supply_Systems/NPS_Water_Systems_Database.xlsx"
+
+
+
+# Supply table (Currently filtered to BRCA)
+supply_table <- read_excel(table_path, sheet = 2, na = "NA") %>%
+  janitor::clean_names() %>%
+  dplyr::filter(region %in% c("Pacific West Region", "Intermountain Region")) %>%
```

Comment:
Is this step needed if you are filtering to just the parks right after?

Reply:
Oops no that's an edit leftover from exploring with Kasper.